pkg/store/store.go

```diff
@@ -126,6 +129,16 @@ func Create(opts ...microstore.Option) microstore.Store {
 				natsjs.NatsOptions(natsOptions), // always pass in properly initialized default nats options
 				natsjs.DefaultTTL(ttl))...,
 		) // TODO test with ocis nats
+	case TypeNatsJSKV:
+		// NOTE: nats needs a DefaultTTL option as it does not support per Write TTL ...
+		ttl, _ := options.Context.Value(ttlContextKey{}).(time.Duration)
+		natsOptions := nats.GetDefaultOptions()
+		natsOptions.Name = "TODO" // we can pass in the service name to allow identifying the client, but that requires adding a custom context option
```

Comment:
TODO?

Reply:
copied from the natsjs store. The point is we would need another envvar `MICRO_STORE_NAME` or similar. Not sure if this is needed and/or wanted. Left it in so we do not forget. I can remove it if you want?

Reply:
Nah, I'm ok with that ;-)